scripts/imageSize.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
+const path = require('path');
+const fs = require('fs');
+
+let fileArray = [];
```

Comment:
Instead of storing it to an external array, I think we can return the result directly by filtering instead of mapping at the end of `read`.  
```
  ...
  .filter(image => fs.lstatSync(image).size > 1000000)
```
Then we can just set `const fileArray = read('src/images/')`.